Project.toml

```diff
@@ -12,7 +12,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 libblastrampoline_jll = "8e850b90-86db-534c-a0d3-1478176c7d93"
 
 [compat]
-METIS4_jll = "400.0.300"
+METIS4_jll = "=400.0.301"
```

Comment:
Is the = intended here?

Reply:
Yes, I always fix the version of the artifacts now.
It will not break packages if the artifact is regenerated.

Reply:
It's what we do with Ipopt_jll: https://github.com/jump-dev/Ipopt.jl/blob/master/Project.toml#L13

Reply:
I always thought we want to minimize the impact of fixing dependencies like this, since it will hold back the resolver from upgrading other packages. I think it is better to leave this open for packages where things are reasonably stable (with some risk of future problems).

Reply:
Yes, you're right @ViralBShah but I find that it's safer to wait that the compat helper workflow opens a PR to add the new version of the artifact.
The only drawback is that we need to do a new release after the update of the Project.toml.

Off-topic: The HSL team is working on a new open-source version `SparseHSL` for information!